ADD: Review Dates to standards and patterns (#66)
OfqualGovUK/ofqual-standards-patterns: docs/patterns/pattern-overview.md

We don't have a standard way of writing patterns, as we like the communities to decide on how best to write and maintain any 'templates' for this, as this allows communities and areas to be a bit more flexible with their structure. Please work with your community to produce your pattern documentation with suitable consistency.

> Any pattern should be reviewed regulalry for relevence (against daily operational norms and technology changes), at least every 6 months is suggested.

Reviewer: ```suggestion
Any pattern should be reviewed regularly for relevance against daily operational norms and technology changes, preferably every 6 months.
```
